migrations/sql/V2019.09.04.11.35__incident_add_mine_determination_columns.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE mine_incident
```

Comment:
Is there already data in mine_incident? Just flagging it to test on a non-empty db as well. FK errors usually get missed till we hit test 😅

Reply:
There is already data in that table. I ran this locally on a pre-seeded database without any issue. The FK is being added to a nullable column that is being created in this migration. So, every record will be `null`, which does not violate the FK constraint.

I think this should be totally safe. Anything you think I may have missed?

Reply:
Nope, looks good 👍